### ISSUE TITLE
no-self-assign error message is bad

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno_lint#666]
```
error[no-self-assign]: length
  --> /Users/lucacasonato/projects/github.com/denoland/deno/op_crates/fetch/22_body.js:80:23
   |
80 |       second.length = second.length;
   |                       ^^^^^^^^^^^^^
   |
   = help: Mistake
```

This error message should be reworked. It need a better help text, and a better error text.

**Comment by kamchy** [comment]
I'll try to look into this issue.

**Comment by magurotuna** [comment]
@bartlomieju @lucacasonato This issue can be closed as it's been resolved by #681.
